README.md

```diff
@@ -1,199 +1,243 @@
-
-```php
-// in a policy
-...
-    ->addDirective(Directive::CAMERA, [
-        Value::SELF,
-        'spatie.be',
-    ])
-    ->addDirective(Directive::GYROSCOPE, 'self spatie.be')
-...
-```
-
-## Creating Policies
-
-The `policy` key of the `feature-policy` config file is set to `Mazedlx\FeaturePolicy\Policies\Basic::class` by default, which allows your site to use a few of the available features. The class looks like this:
-
-```php
-<?php
-
-namespace Mazedlx\FeaturePolicy\Policies;
-
-use Mazedlx\FeaturePolicy\Value;
-use Mazedlx\FeaturePolicy\Directive;
-
-class Basic extends Policy
-{
-    public function configure()
-    {
-        $this->addDirective(Directive::GEOLOCATION, Value::SELF)
-            ->addDirective(Directive::FULLSCREEN, Value::SELF);
-    }
-}
-```
-
-Let's say you're happy with allowing `geolocation` and `fullscreen` but also wanted to add `www.awesomesite.com` to gain access to this feature, then you can easily extend the class:
-
-```php
-<?php
-
-namespace App\Services\FeaturePolicy\Policies;
-
-use Mazedlx\FeaturePolicy\Directive;
-use Mazedlx\FeaturePolicy\Policies\Basic;
-
-class MyFeaturePolicy extends Basic
-{
-    public function configure()
-    {
-        parent::configure();
-
-        $this->addDirective(Directive::GEOLOCATION, 'www.awesomesite.com')
-            ->addDirective(Directive::FULLSCREEN, 'www.awesomesite.com');
-    }
-}
-```
-
-Don't forget to change the `policy` key in the `feature-policy` config file to the class name fo your policy (e.g. `App\Services\Policies\MyFeaturePolicy`).
-
-## Testing
-
-You can run all tests with:
-
-```bash
-$ composer test
-```
-
-## Changelog
-
-Please see [CHANGELOG](https://github.com/mazedlx/laravel-feature-policy/blob/master/CHANGELOG.md) for more information what has changed recently.
-
-## Contributing
-
-Please see [CONTRIBUTING](https://github.com/mazedlx/laravel-feature-policy/blob/master/CONTRIBUTING.md) for details.
-
-### Security
-
-If you discover any security related issues please email mazedlx@gmail.com instead of using the issue tracker.
-
-## Credits
-
-- [Freek van der Herten](https://github.com/freekmurze)
-- [Thomas Verhelst](https://github.com/TVke)
-- [All Contributors](https://github.com/mazedlx/laravel-feature-policy/contributors)
-
-## Support
-
-If you like this package please feel free to star it.
-
-## License
-
-The MIT License (MIT). Please see [LICENSE](https://github.com/mazedlx/laravel-feature-policy/blob/master/LICENSE.md) for more information.
+# Configure the browsers abilities
+
+[![Latest Version on Packagist](https://img.shields.io/packagist/v/mazedlx/laravel-feature-policy.svg?style=flat-square)](https://packagist.org/packages/mazedlx/laravel-feature.policy)
+[![Tests](https://github.com/mazedlx/laravel-feature-policy/actions/workflows/test.yml/badge.svg)](https://github.com/mazedlx/laravel-feature-policy/actions/workflows/test.yml)
```

Comment:
There doesn't seem to be a config present for Travis CI, so I guess it's not used anymore.

In its stead I have added badges from the Github Actions we now have